Dockerfile

```diff
@@ -1,45 +1,60 @@
 FROM ruby:2.3
 
-RUN apt-get update && \
-    apt-get install --no-install-recommends -y cron && \
-    rm -rf /var/lib/apt/lists/* && \
-    apt-get clean
+RUN SUPERCRONIC_URL=https://github.com/aptible/supercronic/releases/download/v0.1.3/supercronic-linux-amd64 && \
```

Comment:
You usually put version numner (0.1.3) and sha1 into Docker ENV variables, which makes them more readable.

Reply:
They are only really needed in this RUN step. I'd consider it litter to leave these variables also set in the final image (e.g. when requesting an interactive shell). Or is that better?